File: src/DZarsky.TS3Viewer2.Web/src/api/models/ClientDetailDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

export type ClientDetailDto = {
    version?: string | null;
    plattform?: string | null;
    description?: string | null;
    inputMuted?: boolean;
    outputMuted?: boolean;
    outputOnlyMuted?: boolean;
    isRecording?: boolean;
    serverGroupIds?: Array<number> | null;
    channelGroupsIds?: Array<number> | null;
    totalConnectionCount?: number;
    away?: boolean;
    isClientTalking?: boolean;
    awayMessage?: string | null;
};

File: src/DZarsky.TS3Viewer2.Domain/Server/Dto/ClientDetailDto.cs
namespace DZarsky.TS3Viewer2.Domain.Server.Dto;

public sealed class ClientDetailDto
{
    public string? Version { get; set; }

    public string? Plattform { get; set; }

    public string? Description { get; set; }

    public bool InputMuted { get; set; }

    public bool OutputMuted { get; set; }

    public bool OutputOnlyMuted { get; set; }

    public bool IsRecording { get; set; }

    public IReadOnlyList<int>? ServerGroupIds { get; set; }

    public IReadOnlyList<int>? ChannelGroupsIds { get; set; }

    public int TotalConnectionCount { get; set; }

    public bool Away { get; set; }

    public bool IsClientTalking { get; set; }

    public string? AwayMessage { get; set; }
}
